src/Cache/Adapter/Filesystem.php

```diff
@@ -93,6 +93,16 @@ public function flush(): bool
         return $this->deleteDirectory($this->path);
     }
 
+    /**
+     * @return bool
+     */
+    public function ping(): bool
+    {
+        $this->save('ping', 'pong');
+
+        return $this->load('ping', 1) === 'pong';
```

Comment:
```suggestion
        $this->save('utopia-ping', 'pong');

        return $this->load('utopia-ping', 1) === 'pong';
```

Reply:
Fixed so we don't need reserved keywords at all.